app/models/client.rb

```diff
@@ -501,11 +500,7 @@ def available_text
         'Fully matched'
       end
     else
-      if parked?
-        "Parked until #{prevent_matching_until}"
-      else
-        'Not available'
-      end
+      'Not available'
```

Comment:
Maybe bring back the if and return`Not available on any route`? It seems like there is a difference between these two cases that might matter to users.

Reply:
There is a difference, but we can no longer include a date because each "parking" can include its own date.

Reply:
Yes, I just wondered if we should return different language so the user might think to look at the routes.. I'm tempted to suggest something like "Not available on any route until {earliest_date}" but that might be misleading...